torch/_dynamo/variables/lists.py

```diff
@@ -44,6 +44,31 @@ def _listlike_contains_helper(items, search, tx, options):
     return result
 
 
+def _index_helper(items, search, tx, args, options):
```

Comment:
I don't see SliceVariable created directly, or indirectly (wrap_slice_range). You might be doing that via BuiltinVariable below but it is not clear to me.